db/init.sql

```diff
@@ -0,0 +1,2 @@
+-- Initialize additional databases for test environment
+CREATE DATABASE vglist_test OWNER vglist;
```

Comment:
This isn't necessary and should be done as part of running `rails db:setup` when setting up the app.